Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -100,12 +103,22 @@ std::optional<IPCReply> BluetoothRealDevice::Open(const OpenRequest& request)
     if (IsBluetoothDevice(descriptor) && IsWantedDevice(device_descriptor) && OpenDevice(device))
     {
       unsigned char manufacturer[50] = {}, product[50] = {}, serial_number[50] = {};
-      libusb_get_string_descriptor_ascii(m_handle, device_descriptor.iManufacturer, manufacturer,
-                                         sizeof(manufacturer));
-      libusb_get_string_descriptor_ascii(m_handle, device_descriptor.iProduct, product,
-                                         sizeof(product));
-      libusb_get_string_descriptor_ascii(m_handle, device_descriptor.iSerialNumber, serial_number,
-                                         sizeof(serial_number));
+      const int manufacturer_ret = libusb_get_string_descriptor_ascii(
+          m_handle, device_descriptor.iManufacturer, manufacturer, sizeof(manufacturer));
+      const int product_ret = libusb_get_string_descriptor_ascii(
+          m_handle, device_descriptor.iProduct, product, sizeof(product));
+      const int serial_ret = libusb_get_string_descriptor_ascii(
+          m_handle, device_descriptor.iSerialNumber, serial_number, sizeof(serial_number));
+      if (manufacturer_ret < LIBUSB_SUCCESS || product_ret < LIBUSB_SUCCESS ||
+          serial_ret < LIBUSB_SUCCESS)
+      {
+        ERROR_LOG_FMT(IOS_WIIMOTE,
+                      "Failed to get descriptor for device {:04x}:{:04x} (rev {:x}): {}/{}/{}",
+                      device_descriptor.idVendor, device_descriptor.idProduct,
+                      device_descriptor.bcdDevice, LibusbUtils::ErrorWrap(manufacturer_ret),
+                      LibusbUtils::ErrorWrap(product_ret), LibusbUtils::ErrorWrap(serial_ret));
+        return true;
+      }
```

Comment:
The new error seems to be this:

```
 56:49:662 Core/IOS/USB/Bluetooth/BTReal.cpp:116 E[IOS_WIIMOTE]: Failed to get descriptor for device 057e:0305 (rev 100): Other error (13: **UNKNOWN**)/Other error (8: **UNKNOWN**)/Invalid parameter (-2: LIBUSB_ERROR_INVALID_PARAM)
```

i.e. `libusb_get_string_descriptor_ascii` is returning an error code, and now we're treating that as a problem while before we were fine with it.